components/shared/Banner/Banner.tsx

```diff
@@ -6,7 +6,7 @@ interface BannerProps {
 const Banner = ({ title, tag }: BannerProps) => {
   return (
     <>
-      <div className="w-96 px-6 py-4 bg-indigo-500 inline-flex justify-start items-center gap-3">
+      <div className="w-full px-6 py-4 bg-indigo-500 inline-flex justify-start items-center gap-3">
         <div className="min-w-0 flex-1 inline-flex flex-col justify-center items-start gap-2">
           <p className="overflow-x-hidden self-stretch justify-center text-Color-white text-base font-bold font-['SUIT_Variable'] leading-tight">
```

Comment:
```suggestion
          <p className="overflow-x-hidden self-stretch justify-center text-Color-white text-base font-bold font-SUIT_Variable leading-tight">
```
`font-SUIT_Variable` 으로만 써도 인식 되는 것 같습니다. 다른 쪽두요!

Reply:
아 넵 맞습니답.

figma에서 코드 복사하면 `font-['SUIT_Varaible']` 로 되길래,
수정하기 싫어서 이대로 적용 되도록 설정해서 크게 문제는 없을 것 같긴 합니다